cli/src/cli.mjs

```diff
@@ -400,11 +402,10 @@ if ( parsed.help ) {
     console.log(ex);
   }
 
-} else if ( parsed.session ) {
-  const record = parsed.session;
-  if ( record === 'start' ) {
+} else if ( parsed.session || parsed.sessionOutput || parsed.sessionOutputPath) {
+  if ( parsed.session && parsed.session == 'start' ) {
     msg(`Starting session...`);
-    axios.post(url(host, port, '/api/v1/session/start'), undefined)
+    await axios.post(url(host, port, '/api/v1/session/start'), undefined)
```

Comment:
I asked why we needed this "await" and this was JB's response: "I changed these to await so that if you call 'sessionOutput' or 'sessionOutputPath' with 'session' start or stop they will work together. The start needs to happen before setting output or path and stop needs to happen afterwards so it needs to be synchronous"